libnavui-voice/src/main/res/layout/mapbox_audio_guidance_button_layout.xml

```diff
@@ -11,9 +11,11 @@
     <androidx.constraintlayout.widget.ConstraintLayout
         android:id="@+id/container"
         android:layout_width="wrap_content"
-        android:layout_height="52dp"
-        android:minWidth="72dp"
-        android:paddingStart="@dimen/mapbox_audioGuidanceButton_paddingStart">
+        android:layout_height="wrap_content"
+        android:layout_gravity="center"
```

Comment:
I don't see the need for `android:layout_gravity="center"`. Work fine even after removing this line.